how-to/taskcluster/new_region.rst

```diff
@@ -0,0 +1,13 @@
+.. new_region:
+
+Adding a new region
+===================
+
+Adding a new GCP region for taskcluster workers requires changes in several places:
+
+- if the workers are going to clone the repository, add clonebundles in that region (e.g. https://bugzilla.mozilla.org/show_bug.cgi?id=1868861)
+- if they're going to run builds, create sccache buckets and service accounts (e.g. https://bugzilla.mozilla.org/show_bug.cgi?id=1882374) and add the service accounts to the taskcluster configuration (e.g. https://mozilla-hub.atlassian.net/browse/SVCSE-2126)
```

Comment:
Similarly, clarifying this comment to be less Gecko-specific would probably be good.